libr/include/r_crypto/r_sm4.h

```diff
@@ -0,0 +1,27 @@
+#ifndef R_SM4_H
+#define R_SM4_H
+
+#include <r_util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Family Key FK */
+static const ut32 FK[4] = { 0xa3b1bac6, 0x56aa3350, 0x677d9197, 0xb27022dc };
+
+/* Constant Key CK */
+const ut32 CK[32] = {
```

Comment:
```suggestion
static const ut32 CK[32] = {
```

Reply:
and move into the .c

Reply:
I agree for FK values but for CK values I also need them in the key search so I need themin the h file.